### PR TITLE
feat-added list_components

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added list Components 
1)Simple List
2)Notification list
3)Imported badge and avatar in components.css
4)removed margin from badge container

### CODE REVIEW

File: componets/lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  padding: 4px;
+}
+
+.list__heading {
+  padding-bottom: 4px;
+  border-bottom: 2px solid var(--border-color);
+}
+.list__container li {
+  list-style: none;
+  margin: 10px;
```
Comment (Prabhav-Sharma) [inline componets/lists/lists.css]: Try to use global variables for properties like margin, padding and gaps, like
--space-xsmall:0.5rem

Comment (prathmeshjagtap) [reply componets/lists/lists.css]: yes I have not declared it globally from starting  so I did not added it

File: componets/lists/lists.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  padding: 4px;
+}
+.list__container li:hover {
+  color: var(--primary-color);
+}
+.list__item_notification {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  border-bottom: 1px solid var(--border-color);
```
Comment (SAYUK09) [inline componets/lists/lists.css]: This isn't noticeable quickly. Maybe increase the intensity.

File: componets/lists/lists.html [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+            you can use this list where you see notifications. Use class
+            <code>.list__item_notification</code> on list items and
+            <code>.list__container</code> on list container
+          </p>
+          <div class="components__example">
+            <ul class="list__container">
+              <li class="list__item_notification">
+                <div class="badge__container">
+                  <img
+                    class="avatar xs"
```
Comment (SAYUK09) [inline componets/lists/lists.html]: maybe we can name this `avatar-xs` for better understanding

File: componets/lists/lists.html [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+                </div>
+                <p>You have new Notification</p>
+              </li>
+              <li class="list__item_notification">
+                <div class="badge__container">
+                  <img
+                    class="avatar xs"
+                    src="../../assets/4.jpg"
+                    href="avatar1"
+                  />
```
Comment (SAYUK09) [inline componets/lists/lists.html]: use `alt` for images always.

**Review by SAYUK09 — CHANGES_REQUESTED** [review]
Nice code. Use some formatting tools like **prettier** while pushing the code, everything else seems good.

File: componets/lists/lists.css (hunk 2 above)
Comment (prathmeshjagtap) [reply componets/lists/lists.css]: I have made changes now its visible.

File: componets/lists/lists.html (hunk 3 above)
Comment (prathmeshjagtap) [reply componets/lists/lists.html]: Did the change of avatar styling now it's more specific.

File: componets/lists/lists.html (hunk 4 above)
Comment (prathmeshjagtap) [reply componets/lists/lists.html]: Added alt tag.

**Pull request merged**